TRADEMARK-COMMITTEE.md

```diff
@@ -0,0 +1,145 @@
+---
+title: "Knative Trademark committee"
+linkTitle: "Trademark committee"
+weight: 50
+type: "docs"
+aliases:
+  - /contributing/trademark-committee/
+---
+
+The Knative Trademark Committee (KTC)
+[advises the trademark owner on the usage and enforcement of the Knative mark](./GOVERNANCE.md).
+It is a separate body from the [Steering Committee](./STEERING-COMMITTEE.md) to
+better represent the needs of vendors and other companies which market products
+in conjuction with the Knative mark.
+
+The governance of Knative is an open, living document, and will continue to
+evolve as the community and project change. We expect over time we will adapt
+the way we run this committee, based on feedback from the community.
+
+- [Charter](#charter)
+- [Delegated Authority](#delegated-authority)
+- [Committee Meetings](#committee-meetings)
+- [Committee Members](#committee-members)
+- [Decision Process](#decision-process)
+- [Getting in touch](#getting-in-touch)
+- [Composition](#composition)
+  - [Maintaining the Trademark Committee](#maintaining-the-trademark-committee)
+
+## Charter
+
+1. Advise the Trademark owner on the definition and enforcement of the "Knative"
+   trademark usage and branding guidelines.
+
+1. Review and approve conformance rules and tests that define brand use
+   decisions.
+
+1. Review and approve the scope and definition of additional derivative Knative
+   marks (e.g. Knative Serving, Knative Core, etc). It is expected that the
+   Trademark Committee will delegate one or more marks to the steering committee
+   to provide branding for nascent efforts which are not yet subject to
+   conformance and third-party licensing.
+
+1. Only the Trademark Committee can redefine its own charter, membership, and
+   authority over Knative Trademark, conformance, and brand decisions.
+
+## Delegated authority
+
+The Trademark Committee may choose to create and delegate its authority to other
+committees as-needed. This may be used (for example) to create sub-committees
+for specific technical or marketing questions related to conformance or joint
+marketing.
+
```

Comment:
Insert: "The Trademark Committee may delegate enforcement authority for one or more marks to the steering committee to provide branding for nascent efforts which are not yet subject to conformance and third-party licensing."

Reply:
Added an additional sentence that this delegation could be further delegated as necessary.